### PR TITLE
Upload image and tags in one api

### PR DESCRIPTION
**Pull request by sudoDeznit**
use new unified api -->  /api/photos/submit-with-tags to upload images

**Note**
- needs some changes on backend for this to work properly.
- require this [Backend PR](https://github.com/OpenLitterMap/openlittermap-web/pull/364) to be merged for this to work.

### CODE REVIEW

File: actions/shared_actions.js [hunk 1]
```diff
@@ -104,86 +104,78 @@ export const toggleSelecting = () => {
...
+            response = await axios(URL + '/api/add-tags', {
                 method: 'POST',
                 headers: {
-                    Authorization: 'Bearer ' + token,
-                    'Content-Type': 'multipart/form-data'
+                    Authorization: 'Bearer ' + token
                 },
-                data: image
+                data: {
+                    litter: tags,
```
Comment (xlcrr) [inline actions/shared_actions.js:119]: keyword should be tags. Need to update the backend too.

File: actions/shared_actions.js [hunk 2]
```diff
@@ -104,86 +104,78 @@ export const toggleSelecting = () => {
...
 
-        console.log('uploadTags', response.data);
-
-        if (response && response?.data?.success) {
+        if (response && response.data?.success) {
+            // return the photo.id that has been created on the backend
             return {
-                success: true
+                success: true,
+                photo_id: response.data.photo_id
```
Comment (xlcrr) [inline actions/shared_actions.js:178]: How are the tags uploaded? Is it being done in 1 request?

Comment (xlcrr) [reply actions/shared_actions.js:178]: I see now in HomeScreen.js

**Review by xlcrr — APPROVED** [review]
Nice work. Keyword should be tags, we need to update the backend too

**Comment by xlcrr** [comment]
We need to start adding documentation to the wiki

**Comment by sudoDeznit** [comment]
Needs to be changed after #129

**Pull request closed without merging**